yb-voyager/src/query/queryparser/helpers_struct.go

```diff
@@ -25,14 +25,10 @@ import (
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
 
-const (
-	LIMIT_OPTION_WITH_TIES = pg_query.LimitOption_LIMIT_OPTION_WITH_TIES
-)
-
 func IsPLPGSQLObject(parseTree *pg_query.ParseResult) bool {
 	// CREATE FUNCTION is same parser NODE for FUNCTION/PROCEDURE
-	_, isPlPgSQLObject := getCreateFuncStmtNode(parseTree)
-	return isPlPgSQLObject
+	node, isPlPgSQLObject := getCreateFuncStmtNode(parseTree)
+	return isPlPgSQLObject && (node.CreateFunctionStmt.SqlBody == nil) //TODO fix proper https://github.com/pganalyze/pg_query_go/issues/129
```

Comment:
how/where are we handling the case when `node.CreateFunctionStmt.SqlBody != nil` in a plpgsql object.

Reply:
There is no need to handle this case because `sqlBody` is only possible in language `sql`, so language `plpgsql` can't have `sql_body` in parser.
docs- https://www.postgresql.org/docs/15/sql-createfunction.html#:~:text=The%20body%20of%20a%20LANGUAGE%20SQL%20function.%20This%20can%20either%20be%20a%20single%20statement